config/peerpods/podvm/agent-policy/coco-policy.rego

```diff
@@ -0,0 +1,39 @@
+package agent_policy
+
+default AddARPNeighborsRequest := true
+default AddSwapRequest := true
+default CloseStdinRequest := true
+default CopyFileRequest := true
+default CreateContainerRequest := true
+default CreateSandboxRequest := true
+default DestroySandboxRequest := true
+default GetMetricsRequest := true
+default GetOOMEventRequest := true
+default GuestDetailsRequest := true
+default ListInterfacesRequest := true
+default ListRoutesRequest := true
+default MemHotplugByProbeRequest := true
+default OnlineCPUMemRequest := true
+default PauseContainerRequest := true
+default PullImageRequest := true
+default ReadStreamRequest := false
```

Comment:
This setting is known to possibly cause [deadlocks](https://github.com/kata-containers/kata-containers/issues/10680) and we already have a related ticket in jira (https://issues.redhat.com/browse/KATA-3440).

I think this should be documented somewhere, like CoCo did for [v0.12.0](https://github.com/confidential-containers/confidential-containers/blob/1a93fb6ccb596ff60f8a8a704fc8c47847526e7b/releases/v0.12.0.md?plain=1#L90).

Reply:
Updated in reference jira